doc/tech/README.md

```diff
@@ -3,203 +3,166 @@
+ - Slower adoption of cloud-native NFV applications and architectures.  A Common Telco Cloud may provide an easier path to methodologies that will drive faster cloud-native NFV application development.
+ - Increased operational overhead due to the need for operators to integrate diverse and sometime conflicting VNF platform requirements.
 
-- Para-Virtualization method (software only).
-- Direct Assignment via h/w assisted PCI-Passthrough (IOMMU).
-- Device Sharing with SR-IOV & h/w assisted PCI-Passthrough (IOMMU).
-- Para-Virtualization method with Hardware support.
+One of major challenges holding back the more rapid and widespread adoption of VNF is that the traditional telecom ecosystem vendors, while building or designing their virtualised services (whether it be Voice over LTE (VoLTE), Evolved Packet Core (EPC), or popular customer facing enterprise services such as SD WAN (Software Defined Wide Area Network), are making their own infrastructure assumptions and requirements, often with custom design parameters. This leaves the operators being forced to build complex integrations of various vendor/function specific silos which are incompatible with each other and might possibly have different and conflicting operating models. In addition, this makes the onboarding and conformance processes of VNFs (coming from different vendors) hard to automate and standardise.  
 
-**Figure 3** below shows some of the relevant IO Virtualization techniques.
+To put this effort in perspective, over the past few years, the telecom industry has been going through a massive technology revolution by embracing software defined networking and cloud architecture principles, in pursuit of the goal of achieving more flexibility, agility and operational efficiency. At a high level, the main objective of NFV (Network Function Virtualisation) is the ability to use general purpose standard COTS (Commercial off the Shelf) compute, memory and storage hardware platforms to run multiple Virtualised Network Functions.  VNFs is the general term that covers any type of virtualised application wither it be in the form of a Virtual Machine (VM) or a containerised application.  Earlier common infrastructure models built on the previous assumption that networking applications are typically built on discrete hardware, do not offer the level of flexibility and agility needed for the support of newer networking technologies such as 5G, intelligent networks and Edge computing.  By running network applications as software rather than on purpose-built hardware, as it has been done since the early 1990’s, the operators aspire to realize operational efficiencies, and capital expense savings.  These Software Defined Network (SDN) applications are increasingly being used by telecom operators to support their internal and customer facing network infrastructures.  The need for a common model across the industry to facilitate more rapid adoption is clear.
```

Comment:
whether :-)

Reply:
I never gets it right :) fixed in: https://github.com/cntt-n/CNTT/pull/1267/commits/5360a5072b790a1beb120203085e58a87e10c385

Reply:
There are still things wrong in the paragraph. 
"Earlier common infrastructure models  ..." which common infrastructure models?

"By running network applications as software rather than on purpose-built hardware, as it has been done since the early 1990’s,  ..." (i) applications are software and (ii) the concept of network elements as software on purpose-built hardware has been around since at least before 1970's -- the 5ESS is a prime example.